tests/integration_msm.rs

```diff
@@ -302,8 +302,10 @@ fn msm_bls12_377_precompute_test() -> Result<(), Box<dyn std::error::Error>> {
 
     log::debug!("Timer generation start");
     let start_gen = Instant::now();
-    let (points, scalars, _, results) =
-        msm::input_generator_bls12_377(Pow::pow(base, max_exp) as usize, true);
+    let (points, scalars, _, results) = msm::input_generator_bls12_377(
+        Pow::pow(base, max_exp) as usize,
+        msm_api::PRECOMPUTE_FACTOR_BASE,
```

Comment:
Should this be the PRECOMPUTE_FACTOR since the line used to contain `true` for precompute?
```suggestion
        msm_api::PRECOMPUTE_FACTOR,
```

Reply:
yeah, you're right, thanks